http/api/transport.go

```diff
@@ -29,8 +29,10 @@ import (
 )
 
 const (
-	protocol    = "http"
-	contentType = "application/senml+json"
+	protocol         = "http"
+	contentTypeSenMLJSON = "application/senml+json"
+	contentTypeSenMLCBOR  = "application/senml+cbor"
+	contentTypeJSON  = "application/json"
```

Comment:
Consider shortening to `ctSenmlJSON`, `ctSenmlCBOR`, `ctJSON` (I know it's `SenML` and not `Senml`, but this looks more natural; and since it's a local variable used only once, `ct` instead of `contentType` sounds acceptable).